[32m2021-06-22 10:53:33[0m - dagster - [34mDEBUG[0m - apis_pipeline - 62b8ca82-0fb8-43c4-a57c-fc3bf4f05439 - 495 - query_wikidata - LOGS_CAPTURED - Started capturing logs for solid: query_wikidata.
[32m2021-06-22 10:53:33[0m - dagster - [34mDEBUG[0m - apis_pipeline - 62b8ca82-0fb8-43c4-a57c-fc3bf4f05439 - 495 - query_wikidata - STEP_START - Started execution of step "query_wikidata".
[32m2021-06-22 10:53:33[0m - dagster - [34mERROR[0m - [31mapis_pipeline - 62b8ca82-0fb8-43c4-a57c-fc3bf4f05439 - 495 - query_wikidata - STEP_FAILURE - Execution of step "query_wikidata" failed.

dagster.core.errors.DagsterExecutionStepExecutionError: Error occurred while executing solid "query_wikidata"::

TypeError: 'NoneType' object is not subscriptable

Stack Trace:
  File "/home/pitanga/.local/share/virtualenvs/situated-views-yLYNqPII/lib/python3.8/site-packages/dagster/core/execution/plan/utils.py", line 42, in solid_execution_error_boundary
    yield
  File "/home/pitanga/.local/share/virtualenvs/situated-views-yLYNqPII/lib/python3.8/site-packages/dagster/utils/__init__.py", line 382, in iterate_with_context
    next_output = next(iterator)
  File "/home/pitanga/.local/share/virtualenvs/situated-views-yLYNqPII/lib/python3.8/site-packages/dagster/core/definitions/decorators/solid.py", line 307, in _coerce_solid_compute_fn_to_iterator
    result = fn(context, **kwargs) if context_arg_provided else fn(**kwargs)
  File "/mnt/d/ims/situated-views/bin/solids/apis.py", line 82, in query_wikidata
    endpoint = context.solid_config["url"]
[0m
